addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -234,7 +234,7 @@ def _l10n_it_get_document_type(self, invoice):
             info_partner_in_eu = infos.get('partner_in_eu', False)
             if all([
                 invoice.move_type in infos.get('move_types', False),
-                invoice._is_downpayment() == infos.get('downpayment', False),
+                infos.get('downpayment') in (None, invoice._is_downpayment()),
```

Comment:
@lordkrandel 'get' falls back to False and not None.  I guess it will result in TD01 instead of TD02 for down payment...

Reply:
![image](https://user-images.githubusercontent.com/1665365/195048618-b4a899c0-223a-48f0-ba0d-5693ca172575.png)

Where did you take this strange idea?

Reply:
but my second comment still stands

Reply:
I thought the two sentences were connected, but they aren't.
Fixin.